packages/usa-combo-box/src/index.js

```diff
@@ -260,11 +258,7 @@ const enhanceComboBox = (_comboBoxEl) => {
         aria-labelledby="${listIdLabel}"
         hidden>
       </ul>
-      <div class="${STATUS_CLASS} usa-sr-only" role="status"></div>
-      <span id="${assistiveHintID}" class="usa-sr-only">
-        When autocomplete results are available use up and down arrows to review and enter to select.
-        Touch device users, explore by touch or with swipe gestures.
-      </span>`,
```

Comment:
**question:** Is this a breaking change if the text was redundant _and_ dynamically generated? Doesn't seem so to me.

Reply:
1. Javascript API

The JavaScript API is the interface controls we expose to users. For example, being able to import combobox by the name combobox and call one of it's methods, like `displayList`

As long as the developers can still access this component and it's methods in the same ways, we aren't changing the JavaScript API.

2. Changes to markup

Changes to markup only include changes to the markup required to implement the component in a project (i.e. [component code](https://designsystem.digital.gov/components/combo-box/#component-code) preview on Site). The aria-instructions are being added dynamically via JS upon initialization, so this change does not affect devs who are implementing this component on their projects.